src/cheri/xcheriot.adoc

```diff
@@ -0,0 +1,397 @@
+:cheriot_ext_name: Xcheriot
+
+:cheriot_auicgp_name: AUICGP
+:cheriot_auicgp_name_lc: auicgp
+
+:cheriot_setbounds_round_down_name: YBNDSRDW
+:cheriot_setbounds_big_imm_name: YBNDSWBI
+
+:cheriot_ot_x_min: 1
+:cheriot_sentry_ot_min: 1
+:cheriot_sentry_ot_bd: 4
+:cheriot_sentry_ot_be: 5
+:cheriot_sentry_ot_fd: 2
+:cheriot_sentry_ot_fe: 3
+:cheriot_sentry_ot_fi: 1
+:cheriot_sentry_ot_max: 5
+:cheriot_ot_x_max: 7
+:cheriot_ot_absent: 8
+:cheriot_ot_nx_min: 9
+:cheriot_ot_nx_max: 15
+:cheriot_ot_max: 15
+
+[appendix]
+[#section_cheriot,reftext="{cheriot_ext_name}"]
+== "{cheriot_ext_name}" Extension and Capabilities
+
+=== Required Extensions
+
+{cheriot_ext_name} is specific to {cheri_base32_ext_name}.
+It requires the presence of all of:
+
+* <<section_priv_cheri,{cheri_priv_m_ext}>>,
+
+* <<section_zylevels1>>,
+
+* <<section_zyseal>>, and
+
+* <<section_zysentry>>.
+
+The present specification presumes the *absence* of all of:
+
+* H-, S-, or U-mode,
+
+* <<section_cheri_hybrid_ext,{cheri_default_ext_name}>>,
+
+* <<section_cheri_disable,{cheri_priv_m_reg_enable_ext}>>,
+
+* <<section_zyseal_amb>>, and
+
+* ...
+
+[NOTE]
+=====
+While {cheriot_ext_name} is nominally compatible with
+<<section_cheri_hybrid_ext,{cheri_default_ext_name}>> and
+<<section_cheri_disable,{cheri_priv_m_reg_enable_ext}>>,
+and particular instantiations may opt to permit disabling CHERI,
+we have not yet found compelling reason to formally specify this composition.
+=====
+
+[NOTE]
+=====
+While {cheriot_ext_name} is nominally compatible with <<section_zyseal_amb>>,
+the operating systems written for {cheriot_ext_name} have security models
+that presume the absence of ambient sealing.
+=====
+
+=== New Instructions
+
+==== Loads and Stores With `x0` Authority
+
+Instructions such as `{load_cap_name_lc} {cd}, n(x0)` would,
+in {cheri_base_ext_name},
+necessarily fault due to the clear {ctag} of the all-zeros `x0`.
+{cheriot_ext_name} defines these instructions
+to use <<mtidc>> as the authority instead.
```

Comment:
I dont think I understand this. Why are you using <<mtidc>> as the authority for cheriot's Load Cap?